velox/exec/Aggregate.cpp

```diff
@@ -39,6 +40,24 @@ AggregateFunctionMap& aggregateFunctions() {
   return functions;
 }
 
+const AggregateFunctionMapBase aggregateFunctions(
+    bool includeCompanionFunctions) {
```

Comment:
It might be simpler to make this parameter skipCompanionFunctions instead of includeCompanionFunctions for better usability